### DISCUSSION TITLE
我想在 toolbar 上增加两个按钮，控制编辑器里字号大小，不知道行不行？

### DISCUSSION THREAD

**Discussion by charlestang** [imzbf/md-editor-v3#403] [Q&A]
我现在实现一款编辑器，可能会全屏使用，但是在 27 寸显示器，感觉字号很小，能否调剂编辑器里的字号的大小呢？

**Comment by imzbf** [comment]
按钮来控制编辑器的字号好像不太行，它在css中预设好的。只能说改覆盖成你想要的样式
